src/main/java/appeng/client/render/items/ToolWirelessTerminalRender.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * This file is part of Applied Energistics 2.
+ * Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved.
+ *
+ * Applied Energistics 2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Applied Energistics 2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
+ */
+
+package appeng.client.render.items;
+
+
+import org.lwjgl.opengl.GL11;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.ItemRenderer;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.IIcon;
+import net.minecraftforge.client.IItemRenderer;
+
+import appeng.api.util.AEColor;
+import appeng.client.texture.ExtraItemTextures;
+import appeng.items.tools.powered.ToolWirelessTerminal;
+
+public class ToolWirelessTerminalRender implements IItemRenderer
+{
+
+    @Override
+    public boolean handleRenderType( ItemStack item, ItemRenderType type )
+    {
+        return true;
+    }
+
+    @Override
+    public boolean shouldUseRenderHelper( ItemRenderType type, ItemStack item, ItemRendererHelper helper )
+    {
+        return false;
+    }
+
+    @Override
+    public void renderItem( ItemRenderType type, ItemStack item, Object... data )
+    {
+        Entity itemLocation = item.getItemFrame();
+        if( itemLocation==null )
+        {
+            itemLocation = Minecraft.getMinecraft().thePlayer;
+        }
+        
+        boolean displayAntenna = ((ToolWirelessTerminal)item.getItem()).getIsUsable( item, itemLocation);
+        boolean hasPower = (( ToolWirelessTerminal) item.getItem()).hasPower( null,0.5,item );
+
+        IIcon border;
+        if( displayAntenna )
+        {
+            border = ExtraItemTextures.WirelessTerminal_Border.getIcon();
+        }
+        else
+        {
+            border = ExtraItemTextures.WirelessTerminal_Border_Inactive.getIcon();
+        }
+        IIcon scrollBar = ExtraItemTextures.WirelessTerminal_ScrollBar.getIcon();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'scrollBar' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)